server/athenian/api/models/state/versions/69aed10c7ce1_goals.py

```diff
@@ -0,0 +1,117 @@
+"""goals
+
+Revision ID: 69aed10c7ce1
+Revises: 38ba0c8ae959
+Create Date: 2022-05-02 10:45:57.268703+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "69aed10c7ce1"
+down_revision = "38ba0c8ae959"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "goals",
+        sa.Column(
+            "created_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("account_id", sa.Integer(), nullable=False),
+        sa.Column("template_id", sa.Integer(), nullable=False),
+        sa.Column("valid_from", sa.TIMESTAMP(timezone=True), nullable=False),
+        sa.Column("expires_at", sa.TIMESTAMP(timezone=True), nullable=False),
+        sa.ForeignKeyConstraint(["account_id"], ["accounts.id"], name="fk_goal_account"),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_table(
+        "team_goals",
+        sa.Column(
+            "created_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column("goal_id", sa.BigInteger().with_variant(sa.Integer(), "sqlite"), nullable=False),
+        sa.Column("team_id", sa.Integer(), nullable=False),
+        sa.Column("target", sa.JSON(), nullable=False),
+        sa.ForeignKeyConstraint(["goal_id"], ["goals.id"], name="fk_team_goal_goal"),
+        sa.ForeignKeyConstraint(["team_id"], ["teams.id"], name="fk_team_goal_team"),
+        sa.PrimaryKeyConstraint("goal_id", "team_id"),
+    )
+    op.drop_column("account_features", "created_at")
```

Comment:
are these expected ? Should I remove them from migration ?

Reply:
I found out the sqlite in 20.04 is too old to drop columns, so I'm removing these